### PR TITLE
🔨 Switch Project to use useOvermind

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

**Comment by MichaelDeBoey** [comment]
Typecheck errors will be fixed after #2566 I think

**Comment by christianalfoni** [comment]
Cool, let us update this after we merge the other into master

**Comment by christianalfoni** [comment]
@MichaelDeBoey Okay, cool, try to update this with latest master and see if we can get it through :)

**Comment by MichaelDeBoey** [comment]
@christianalfoni Types are fixed 🙂

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/Project/Project.tsx [hunk 1]
```diff
@@ -36,159 +36,149 @@ import { Keywords } from './Keywords';
...
+            url={githubRepoUrl(sandbox.git)}
+            username={sandbox.git.username}
+            repo={sandbox.git.repo}
+            branch={sandbox.git.branch}
+          />
+        </Item>
+      )}
+
+      <StatsContainer>
+        <Stats />
```
Comment (christianalfoni) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/Project/Project.tsx]:
```suggestion
        <Stats sandbox={sandbox} />
```

This one needs the sandbox :)

**Comment by christianalfoni** [comment]
Sorry, just merged same refactor from Hacktober 😞 Thanks a bunch though!

**Comment by MichaelDeBoey** [comment]
No problem 🙂

**Pull request closed without merging**